Makefile

```diff
@@ -24,11 +24,11 @@ dependencies:
 	glide --version || go get -u -f github.com/Masterminds/glide
 	glide install
 	@echo "Installing test dependencies..."
-	go install ./vendor/github.com/axw/gocov/gocov
-	go install ./vendor/github.com/mattn/goveralls
+	go get -u github.com/axw/gocov/gocov
+	go get -u github.com/mattn/goveralls
 ifdef SHOULD_LINT
 	@echo "Installing golint..."
-	go install ./vendor/github.com/golang/lint/golint
+	go get -u github.com/golang/lint/golint
```

Comment:
I'd like to keep these pinned with Glide, rather than installing whatever's on master. It's very irritating to have builds start failing if upstream breaks or introduces new lint rules.

Concretely, this means that `goveralls` and `golint` should be in `glide.yaml`.